tests/evm/test_chainid.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from zkevm_specs.evm import (
+    ExecutionState,
+    StepState,
+    verify_steps,
+    Tables,
+    RWTableTag,
+    RW,
+    Block,
+    Bytecode,
+    RWDictionary,
+)
+from zkevm_specs.util import rand_fq, RLC, U256
+from zkevm_specs.util.param import N_BYTES_WORD
+
+
+TESTING_DATA = (0x030201, rand_fq().n)
```

Comment:
This is a small detail, but using `rand_fq()` for a chain_id value makes me think that chai_id is in Fq, but in reality it's in Word.  What about using `rand_word()` here?

Reply:
Hmmm that's true. 
And the RLC will be performed anyways! I like it!